ucsb_ds_capstone_projects_2021/projects/sansum/update2.md

```diff
@@ -16,114 +16,148 @@ jupyter:
+
+- Five Determinants of Health: Genetic, Biological, Behavioral, Psychological, and Societal/Environmental factors.
+
+Background
+
+Diabetes is the fifth leading cause of death among Latino Americans. Almost 1 in 2 Latino adults are at risk of type 2 diabetes. Recent data from the CDC shows that diabetes rates are twice as high for Latinos of Mexican heritage compared with Whites. This excess burden of diabetes is due to complex, poorly-understood interactions between determinants of health. Some of these include economic challenges, cultural/language/health literacy barriers, pollution, etc. Many of these factors are beyond traditional biological factors; by learning more about these relationships we can identify interventions that can target diabetes outside of traditional clinical options.
+
+
+
+
+
+### Analysis Strategy
+
+We trichotomized individuals by HbA1c levels. These classes represent an individual's risk of complications from diabetes, based off recommendations from the American Diabetes Association. 
+
+|HbA1c Class|HbA1c levels|Risk level|Description|
+|-----------|------------|----------|:----|
+|     0     |      <7%   | Low risk |No diabetes, pre-diabetes, or diabetics who are able to control their a1c levels with insulin      |
+|1|7-9%|Moderate risk|Poor control over diabetes; at risk for complications with heart, kidney, and eyes|
+|2|>9%|High risk|A1c levels are critically high; Undiagnosed diabetes patients or when current diabetes management is inadequate|
+
+Through data visualizations and linear regressions, we attempted to discover a relationship between features and categorical HbA1c class or the numerical HbA1c level. 
 
 
-<img src = "https://raw.githubusercontent.com/lawrencedlin/ucsb-ds-capstone-2021.github.io/main/ucsb_ds_capstone_projects_2021/projects/sansum/visuals/survey.png">
 
-These plots show the differences between the correlation and stratification of A1c in relation to both the FACIT-Sp-12 total score and subscores. FACIT-Sp-12 stands for “Functional Assessment of Chronic Illness Therapy - Spiritual Well-Being 12 Item Scale”. There are three subscores when using the 3-Factor model: meaning, peace and faith. Higher scores imply better spiritual well being. The meaning subscore has a higher correlation coefficient then the other plots at around ~0.16. The total score has a correlation coefficient of ~0.10 and peace subscore has a correlation coefficient of around ~0.08. The faith subscore, in contrast, has a correlation coefficient as low as ~.01.
+
+### Exploratory Data Analysis
 
 
 <img src = "https://raw.githubusercontent.com/lawrencedlin/ucsb-ds-capstone-2021.github.io/main/ucsb_ds_capstone_projects_2021/projects/sansum/visuals/pca.png" width = 200px, height = 200px>

```

Comment:
Markdown can embed images directly: https://jupyterbook.org/content/figures.html
For the image `visuals/pca.png`, replace the HTML with `![some-text](visuals/pca.png)`